cpp/common/src/codingstandards/cpp/deadcode/UnusedFunctions.qll

```diff
@@ -75,7 +75,9 @@ module UnusedFunctions {
    */
 
   private class MainLikeFunctionEntryPoint extends EntryPoint, MainLikeFunction {
-    MainLikeFunctionEntryPoint() { this instanceof MainLikeFunction }
+    MainLikeFunctionEntryPoint() {
+      this instanceof MainLikeFunction or this instanceof GTestFunction
```

Comment:
`MainLikeFunction` is an abstract class, which means it represents the set of all values from all subclasses already, so you don't need to specifically mention GTestFunction here.
```suggestion
      this instanceof MainLikeFunction
```

Reply:
thanks for noticing! I had missed to modify it after I made GTestFunction extend from MainLikeFunction.